cypress/e2e/Mobile_Check.js

```diff
@@ -0,0 +1,42 @@
+// Mobile_Check.js
+// This test aims to test what a user will encounter if they
+// attempt to complete the experiment on a mobile device
+
+describe("normal_paths", () => {
+  const playerKeys = ["test_" + Math.floor(Math.random() * 1e13)];
+
+  before(() => {
+    cy.empiricaClearBatches();
+    cy.empiricaCreateBatch("cypress1");
+
+    //Start batch
+    cy.get("tr", { log: false })
+      .last({ log: false })
+      .contains("Start", { log: false })
+      .click({ log: "Start Button" });
+    //Check started
+    cy.waitUntil(() =>
+      cy
+        .get("tr")
+        .last()
+        .then(($tr) => $tr.find('button:contains("Stop")').length == 1)
+    );
+  });
+
+  it("redirects to mobile error screen", () => {
+    cy.wrap(playerKeys, { log: false }).each((playerKey) => {
```

Comment:
we only have 1 player in this test, so you can remove line 27 `cy.wrap(playerKeys, { log: false }).each((playerKey) => {` and it's trailing brace, and just have the inside code.